domains/ariez.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "AriezzYT",
+        "email": "ttkeaton@outlook.com"
+      },
+
+      "record": {
+        "A": ["185.199.108.153 185.199.109.153 185.199.110.153 185.199.111.153"]
```

Comment:
Change this to a CNAME: ariezzyt.github.io

Reply:
Updated should be looking ok now.